src/main/java/org/opentripplanner/api/resource/Routers.java

```diff
@@ -213,13 +215,17 @@ public Response postGraphOverWire (
         try {
             graph = Graph.load(is, level);
             GraphService graphService = otpServer.getGraphService();
+            ObjectMapper mapper = new ObjectMapper();
+
+            JsonNode routerJsonConf = mapper.readTree(graph.routerConfig);
+            graphService.registerGraph(routerId, new MemoryGraphSource(routerId, graph, routerJsonConf));
```

Comment:
Thinking again about it, it would be better to put this code directly in the `MemoryGraphSource` constructor, and remove the json config parameter (as the config used is embedded inside the graph which is also part of the arguments). That way it can be re-used by other code, notably the "build graph over wire" end-point (posting a zip).